### DISCUSSION TITLE
v0.11.0

### DISCUSSION THREAD

**Discussion by 1over137** [FreeLanguageTools/vocabsieve#116] [Announcements]
## New features
- Clipboard monitoring should now work on Wayland and on MacOS now that a workaround (polling) has been implemented.
- Word scores are now cached and is now used to display word status on lookup
- Dictionary definition handling has been revamped. Now you can switch between several different results in each of the two definition boxes.
  - You can specify the order in which definitions are fetched, as well as what form of the word to use for each dictionary (lemma policy)
- Pronunciation: Forvo fetching should now be able to get either mp3 or ogg. Windows or IOS users should use mp3.
- Logging is introduced in the program. You can view the logs by going to Help -> View session logs. Please include the log when reporting an issue.

## Other changes
- Interface has been somewhat simplified to reduce clutter. Lookup buttons have been removed but they are still accessible via Ctrl/Cmd+D and Ctrl/Cmd+Shift+D. 
- Shortcut for clearing the selected image is now Ctrl+W in order to ensure that all shortcuts are accessible with the left hand alone. In the future a different set of shortcuts will be added for right-handed operation.
- The local HTTP API has been removed for now. A more refined API will be introduced when the need arises.
- You are recommended to use the new [manual](https://docs.freelanguagetools.org/) from now on.

<hr /><em>This discussion was created from the release <a href='https://github.com/FreeLanguageTools/vocabsieve/releases/tag/v0.11.0'>v0.11.0</a>.</em>

**Comment by andrewweatherly** [comment]
Looks like this release is a bit broken on mac. The configure button isn't along the top bar, so you can't do any of the configuring on macs.
